roles/generate-jenkins/templates/Jenkinsfile.j2

```diff
@@ -451,6 +464,7 @@ pipeline {
                 cd ${TEMPDIR}/docker-${CONTAINER_NAME}
                 mkdir -p ${TEMPDIR}/repo/${LS_REPO}/.github/workflows
                 mkdir -p ${TEMPDIR}/repo/${LS_REPO}/.github/ISSUE_TEMPLATE
+                mkdir -p ${TEMPDIR}/repo/${LS_REPO}/root/etc/cont-init.d
```

Comment:
Should this be in an if statement based on deprecation?

Reply:
Resolved in https://github.com/linuxserver/docker-jenkins-builder/pull/101/commits/8bd329270200113449a568e4ca3970d97d5b9937